packages/hash/datastore/postgres/util.ts

```diff
@@ -1,2 +1,31 @@
+import { Uuid4 } from "id128";
+import { MigrationBuilder } from "node-pg-migrate";
+
+/**
+ * @param db
+ * @param params.tableName the name of the column's table
+ * @param params.columnName the name of the column
+ * @returns `true` if the column exists in the table, otherwise `false`
+ */
+export const columnDoesNotExists = async (
+  db: MigrationBuilder["db"],
+  params: {
+    tableName: string;
+    columnName: string;
+  },
+): Promise<Boolean> => {
+  const { rows } = await db.query(`
+    select * 
+    from INFORMATION_SCHEMA.COLUMNS 
+    where table_name = '${params.tableName}'
+    and column_name = '${params.columnName}'
+  `);
+
+  return rows.length === 0;
+};
```

Comment:
I didn't end up having to use this method, but think it might come in handy down the line so think it should be included in this PR.